numerical_methods/durand_kerner_roots.c

```diff
@@ -108,8 +108,8 @@ int main(int argc, char **argv)
     }
 
     degree = argc - 1; /* detected polynomial degree */
-    coeffs = (double *)malloc(
-        degree * sizeof(double)); /* store all input coefficients */
+    coeffs = (long double *)malloc(
+        degree * sizeof(long double)); /* store all input coefficients */
```

Comment:
printf might complain now

Reply:
oh LGTM issued the alert

Reply:
yes, converted to draft. dont review until marked as ready for review.